### ISSUE TITLE
bxt_ch_infinite_ammo (0|1)

### ISSUE THREAD

**Issue by Droooga** [YaLTeR/BunnymodXT#366]

**Comment by SmileyAG** [comment]
Issue with infinite ammo is that those `bxt_ch_` that are currently available (angles, health, armour, origin, velocity) are taken from the engine (**hw.dll**), so it is easy for us to maintain compatibility for all GoldSrc modifications, since across of them only game DLLs can changed (example: **client.dll** and **hl.dll**).

Ammo values cannot be obtained from the engine  (**hw.dll**), since they are stored in the server-side (example: **hl.dll**), which means that we will have to support absolutely for every modification that command.

TLDR: not worth the trouble, better to use `give` command for obtain needed weapon ammo, @YaLTeR, please close this issue.

**Comment by YaLTeR** [comment]
I agree it's likely more effort than it's worth, but I also thought the same about BXT HUD recoloring, and then it got implemented (and which also heavily relies on client.dll hooks and overrides). So yeah, not sure closing the issue is warranted.
